new-backend/app/modules/langgraph_nodes/fact_check.py

```
@@ -0,0 +1,12 @@
+# web search + fact  check
+
+def search_web():
+    return None
+
+
+def run_fact_check(state):
+    text = state["text"]
+    keywords = state["keywords"]
+    results = search_web(text + " " + " ".join(keywords))
```

Pylint (3.3.7)

[error] Assigning result of a function call, where the function returns None

(E1128)